manifests/reverse_proxy.pp

```diff
@@ -0,0 +1,50 @@
+#Adds http reverse-proxy to parent conf
+class capsule::reverse_proxy (
+
+  $path = '/',
+  $url  = "https://${capsule::parent_fqdn}/",
+  $port = $capsule::params::parent_reverse_proxy_port
```

Comment:
This will always pull in the default value, not the value in `capsule`. 

If this class will always be wrapped by `capsule`, I wouldn't even parameterize, especially since you're "building" a URL in the class definition, I think that doesn't belong there.